### PR TITLE
Adds tests for Django 3.0

### PR DESCRIPTION
**Pull request by mbi**
Replaces import of `django.utils.six`([removed in Django 3.0](https://docs.djangoproject.com/en/dev/releases/3.0/#removed-private-python-2-compatibility-apis)) with the six library.

### CODE REVIEW

File: tests/test_core.py [hunk 1]
```diff
@@ -533,9 +510,7 @@ def render_tag(self, context, values, varname):
                 return value
 
         origtpl = template.Template(
-            '{% for thing in sequence %}'
-            '{% cycle "1" "2" "3" "4" %}'
-            '{% endfor %}'
+            '{% for thing in sequence %}' '{% cycle "1" "2" "3" "4" %}' '{% endfor %}'
```
Comment (karyon) [inline tests/test_core.py]: that didn't exactly improve :)

**Comment by karyon** [comment]
Hi! https://github.com/django-compressor/django-compressor uses django-classy-tags through django-sekizai. it would make things easier for us if classy-tags would support django 3.0. therefore my unsolicited review above :) also, if you don't want to merge this as it is, I could split this PR into the functional parts (and add python 3.8 along the way) and the reformatting.

**Comment by Bachstelze** [comment]
Who has a write access and can make a review?

**Comment by yakky** [comment]
Thanks, I merged all the Django 3 contributions into #50 to ensure a smoother merge process

**Comment by mbi** [comment]
Awesome, closing this and looking forward to #50 then ;-)

**Comment by yakky** [comment]
@mbi thanks for your contributiion. Django 3 support has been merged with #50

**Pull request closed without merging**